ui/src/message_popup/experience_page.jsx

```diff
@@ -159,16 +172,19 @@ export default React.createClass({
 
   renderInstructions() {
     const {scaffolding} = this.state;
+    const {query} = this.props;
+
     return (
       <VelocityTransitionGroup enter={{animation: "callout.pulse", duration: 500}} leave={{animation: "slideUp"}} runOnMount={true}>
-        <div>
+        <div className="instructions">
```

Comment:
@david-rosales This isn't ideal but since I couldn't get the `mount` deep renderer working yet, this is how you can add in classNames to check that the output of render is what you'd expect.